package.json

```diff
@@ -31,6 +31,7 @@
     "@date-io/date-fns": "^1.3.13",
     "@koale/useworker": "^3.4.0",
     "@material-ui/core": "^4.11.2",
+    "@material-ui/icons": "^4.11.2",
```

Comment:
Can you please revert this change? We have already discussed multiple times to use `<Icon>icon_name_here</Icon>` instead of importing specific icons from this package. [If you need to find a specific icon, use the MaterialUI Icon Font page here.](https://fonts.google.com/icons?selected=Material+Icons)

Reply:
@avaidyam We tried to use the Icon. But it is not available there. I tried using that, the same icon was not available. That is why I used this.

Reply:
If I can use this code - <span class="material-icons-outlined">account_circle</span>, I can remove that now. outlined icons are not  rendered. I tried <Icon>account_circle_outlined</Icon>

Reply:
@sarithapillai8 [It looks like it is there though?](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aaccount_circle)

Reply:
@sarithapillai8 That should be fine, to just use `account_circle` instead.